listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -1,3 +1,39 @@
+# This script is responsible to process user listening history and create and save dataframes to be used
+# in training the model. The general flow is as follows:
+#
+# **get_dates_to_train_data: Get timestamp (from_date, to_date) to fetch user listening history.
```

Comment:
This seems more like function documentation, I'd prefer if we modeled them as steps in simple english.